riscv/src/runtime.rs

```diff
@@ -366,9 +498,79 @@ impl TryFrom<&[&str]> for Runtime {
             }
             match *name {
                 "poseidon_gl" => runtime = runtime.with_poseidon(),
+                "arith" => runtime = runtime.with_arith(),
                 _ => return Err(format!("Invalid co-processor specified: {name}")),
             }
         }
         Ok(runtime)
     }
 }
+
+/// Helper function for register names used in instruction params
+fn reg(mut idx: u8) -> String {
+    // s* callee saved registers
```

Comment:
s?

Reply:
`s?`?

Reply:
I meant what's `s*`?

Reply:
register names `s0..=11`

Reply:
I'm referring the risc-v ABI register names here